src/styles/pages/u.module.scss

```diff
@@ -0,0 +1,81 @@
+@use 'src/styles/vars.scss' as vars;
+
+.profilePage {
+  margin: auto;
+  max-width: 960px;
+  position: relative;
+  width: 70%;
```

Comment:
I think we might be able to do
```
width: 100%;
max-width: 768px; (use the actual variable for breakpointMd)
```
to achieve a similar effect on desktop with the column width, but easily transition to a full page column view on mobile. let me know what you think about this ^

Reply:
fixed in Clean up code 1